networkx/generators/classic.py

```diff
@@ -891,15 +930,15 @@ def trivial_graph(create_using=None):
 
 @nx._dispatchable(graphs=None, returns_graph=True)
 def turan_graph(n, r):
-    r"""Return the Turan Graph
+    r"""Return the Turán graph.
 
-    The Turan Graph is a complete multipartite graph on $n$ nodes
+    The Turán graph is a complete multipartite graph on $n$ nodes
     with $r$ disjoint subsets. That is, edges connect each node to
     every node not in its subset.
 
     Given $n$ and $r$, we create a complete multipartite graph with
-    $r-(n \mod r)$ partitions of size $n/r$, rounded down, and
-    $n \mod r$ partitions of size $n/r+1$, rounded down.
+    $r-(n \mod r)$ partitions of size $\lfloor n/r \rfloor$, and
+    $n \mod r$ partitions of size $\lfloor n/r \rfloor + 1$.
```

Comment:
I've probably read more raw TeX math than the vast majority of people, so this is a definite blind spot for me! I try to be careful about introducing TeX, especially in docstrings that don't already contain math-mode expressions. I thought it made sense here to add it because 1. the previous expression was technically wrong and 2. the floor commands are fairly readable in this particular situation, in my opinion.